cloud/provider.go

```diff
@@ -57,6 +55,11 @@ type Node struct {
 	GPUCost          string `json:"gpuCost"`
 }
 
+// IsSpot determines whether or not a Node uses spot by usage type
+func (n *Node) IsSpot() bool {
+	return strings.Contains(n.UsageType, "spot") || strings.Contains(n.UsageType, "emptible")
```

Comment:
@dwbrown2 I pulled this logic from the FE. Does that seem right here?

Reply:
@AjayTripathy isn't there one more test that can be done on the backend? Check spot_label in modelConfigs?